services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -136,6 +136,11 @@ def find_by_permit_id(cls, _id):
     def find_by_now_application_guid(cls, _id):
         return cls.query.filter_by(now_application_guid=_id).first()
 
+    @classmethod
+    def find_original_permit_amendment_by_permit_guid(cls, _guid):
+        return cls.query.filter_by(permit_guid=_guid).filter_by(
+            permit_amendment_type_code='OGP').first()
+
```

Comment:
This may return multiple results in situations where a permit is used by multiple mines (each mine,permit pair will have an OGP).